### DISCUSSION TITLE
Format Option for Dock Plugin "Datetime"

### DISCUSSION THREAD

**Discussion by AcidRain9** [linuxdeepin/developer-center#3059] [Features Request & Ideas | 特性请求 & 头脑风暴]
**Current:**
**YYYY/MM/DD**
![Screen Capture_select-area_20210325135921](https://user-images.githubusercontent.com/26188786/112485815-397a3d80-8d73-11eb-8049-270a8da158ba.png)

**Proposal:**
**DD/MM/YYYY**
![Screen Capture_select-area_20210325135921(2)](https://user-images.githubusercontent.com/26188786/112487098-5feca880-8d74-11eb-8b0b-307888c1aa3f.jpg)

**Comment by XMuli** [comment]
In yyyy-mm-dd is one that is more in line with Chinese customs; however your suggestion could be another option

**Comment by AcidRain9** [comment]
> In yyyy-mm-dd is one that is more in line with Chinese customs; however, your suggestion could be another option

Yeah, it's an extra option to switch to dd/mm//yyyy

**Comment by 0yanghongwei** [comment]
@AcidRain9 We plan to add your suggestion in a later version. Thank you for your suggestion.

**Comment by meiyixiang** [comment]
Hello, thank you for your suggestion. We will consider it, but the display habit of the area
